src/main/java/com/solvd/block2/mybatis/AccountService.java

```diff
@@ -0,0 +1,70 @@
+package com.solvd.block2.mybatis;
+
+
+import com.solvd.block2.mappers.AccountMapper;
+import com.solvd.block2.sql.models.Account;
+import org.apache.ibatis.io.Resources;
+import org.apache.ibatis.session.SqlSession;
+import org.apache.ibatis.session.SqlSessionFactory;
+import org.apache.ibatis.session.SqlSessionFactoryBuilder;
+
+import java.io.InputStream;
+import java.util.List;
+
+public class AccountService {
+    private final SqlSessionFactory sqlSessionFactory;
+
+    public AccountService() {
+        this.sqlSessionFactory = createSqlSessionFactory();
+    }
+
+    // Create session factory
+    private SqlSessionFactory createSqlSessionFactory() {
```

Comment:
Maybe create your session factory as a different class, and just call that class in here. That way, if you were to change the `mybatis_config.xml` path or name, you dont need to go through your entire project changing the path name. Have it as a constant in a different class.